### PR TITLE
Reset column order for lists when using "Reset to Default" action

### PR DESCRIPTION
**Pull request by mjauvin**

### CODE REVIEW

File: modules/backend/widgets/Lists.php [hunk 1]
```diff
@@ -1685,6 +1685,7 @@ public function onApplySetup()
      */
     public function onResetSetup()
     {
+        $this->putSession('order', null);
```
Comment (LukeTowers) [inline modules/backend/widgets/Lists.php]: Can we just make the order a userPreference same as the other ones? Any reason why this is managed a different way than the other ones?

Comment (mjauvin) [reply modules/backend/widgets/Lists.php]: No idea why, I can convert to a user preference if you want.

**Review by bennothommo — APPROVED** [review]
Looks good to me. Happy to merge if it's been tested.

**Comment by mjauvin** [comment]
Yes, it's working great!

**Pull request merged**